classes/OrderManager.php

```diff
@@ -190,10 +190,8 @@ public function saveOrder($order, array $data)
 
         $order->addOrderMenus(Cart::content()->all());
         $order->addOrderTotals($this->getCartTotals());
-
-        // Lets log the coupon so we can redeem it later
-        if ($couponCondition = Cart::conditions()->get('coupon'))
-            $order->logCouponHistory($couponCondition, $this->customer);
+        
+        Event::fire('igniter.checkout.afterSaveOrder', [$order, $this->customer]);
```

Comment:
customer should be available on $order->customer

Reply:
Yeah I had thought that. Was just copying what was there before but makes sense to reduce what is passed.